exercises/concept/freelancer-rates/.meta/exemplar.cpp

```diff
@@ -0,0 +1,35 @@
+// INFO: Headers from the standard library should be inserted at the top via
+#include <cmath>
+
+// daily_rate calculates the daily rate given an hourly rate
+double daily_rate(double hourly_rate) {
+    auto per_hour{8.0};
```

Comment:
I think it'd be better to call this variable `hours_per_day` or something similar; it's not really a per-hour value.